packages/docs/docs/docs/deploying.md

```diff
@@ -87,112 +91,73 @@ contract MyContract is Initializable {
-development of ZeppelinOS compatible smart contracts. In this case, we are importing
-`Initializable.sol` in order to use the `initializer` modifier, which will
-prevent the `initialize` function from be called more than once.
+Notice that the sample contract has an `initialize` function instead of the standard
+Solidity constructor. This is how you define constructor functionality for upgradeable
+contracts in ZeppelinOS, which you'll see in more detail in the next tutorial.
 
-Now we can add the contract to the project:
+Also note that the contract imports another contract from the `zos-lib` package, which
+you installed previously.
 
-```console
-npx zos add MyContract
-```
-
-This command will first compile `MyContract`, and then add it to the
-project, registering it in the `zos.json` configuration file.
-
-> **Note**: The current version of ZeppelinOS relies on Truffle to compile contracts and
-> determine the Solidity compiler version to use. If you want to use a different version, please specify it
-> in the Truffle configuration file.
-> More info:
-> https://github.com/trufflesuite/truffle/releases/tag/v5.0.0-beta.0#specify-a-solcjs-version
 
 ## Deploying your project
 
-And just like that, we are now ready to make the initial deployment of the
-project. We are just missing a blockchain network where it will be deployed.
-For this example, let's use [ganache](https://truffleframework.com/docs/ganache/quickstart),
-a personal blockchain for Ethereum development that you can use to develop
-your contracts. To install it run:
+You're now ready for the initial deployment of the project. To do this, however, you'll need a blockchain network on which to deploy it. For this example, you can use [Ganache](https://truffleframework.com/docs/ganache/quickstart), which is a personal blockchain for Ethereum development that you can use to deploy your contracts. To install it, run:
 
 ```console
 npm install -g ganache-cli
 ```
 
-To start working with it, open a separate terminal and run:
+Open a separate terminal and run Ganache:
 
 ```console
 ganache-cli --port 9545 --deterministic
 ```
 
-Once we have done that, let's go back to the original terminal and
-run the following command:
+Now run `zos create`:
 
 ```console
-npx zos session --network local --from 0x1df62f291b2e969fb0849d99d9ce41e2f137006e --expires 3600
+$ npx zos create
 ```
 
-The `session` command starts a session to work with a desired network.
-In this case, we are telling it to work with the `local` network with the
-`--network` option, and also setting a default sender address for the
-transactions we will run with the `--from` option. Additionally, the
-`expires` flag allows us to indicate the session expiration time in seconds.
+Most of the ZeppelinOS commands, such as `create`, are interactive as of version 2.3 and will prompt you for the needed information. In this case, the `create` command will prompt you, and you can use **Enter** to select the defaults as follows:
```

Comment:
Can we rephrase to avoid the repetition of _will prompt you_, which doesn't add much?

Reply:
Yes, that did sound repetitive. Changed to read:
"Most of the ZeppelinOS commands, such as `create`, are interactive as of version 2.3 and will request information as needed. In this case, use **Enter** to select the defaults as follows;"